packages/dids/package.json

```diff
@@ -76,41 +76,36 @@
   "dependencies": {
     "@decentralized-identity/ion-pow-sdk": "1.0.17",
     "@decentralized-identity/ion-sdk": "1.0.1",
+    "@noble/hashes": "^1.3.2",
+    "bittorrent-dht": "^11.0.5",
+    "bittorrent-dht-sodium": "^1.2.0",
     "@web5/common": "0.2.1",
     "@web5/crypto": "0.2.1",
     "canonicalize": "2.0.0",
     "did-resolver": "4.1.0",
+    "dns-packet": "^5.6.1",
     "level": "8.0.0",
-    "ms": "2.1.3"
+    "ms": "2.1.3",
+    "pkarr": "^1.1.1",
+    "z32": "^1.0.1"
   },
   "devDependencies": {
     "@playwright/test": "1.36.2",
     "@types/chai": "4.3.6",
     "@types/chai-as-promised": "7.1.5",
-    "@types/eslint": "8.44.2",
+    "@types/dns-packet": "^5.6.1",
     "@types/mocha": "10.0.1",
     "@types/sinon": "10.0.15",
-    "@typescript-eslint/eslint-plugin": "6.4.0",
-    "@typescript-eslint/parser": "6.4.0",
+    "buffer": "6.0.3",
     "c8": "8.0.1",
     "chai": "4.3.10",
     "chai-as-promised": "7.1.1",
     "esbuild": "0.16.17",
     "eslint": "8.47.0",
-    "eslint-plugin-mocha": "10.1.0",
     "karma": "6.4.1",
-    "karma-chai": "0.1.0",
-    "karma-chrome-launcher": "3.1.1",
-    "karma-esbuild": "2.2.5",
-    "karma-firefox-launcher": "2.1.2",
-    "karma-mocha": "2.0.1",
-    "karma-mocha-reporter": "2.2.5",
-    "karma-webkit-launcher": "2.1.0",
     "mocha": "10.2.0",
-    "playwright": "1.36.2",
     "rimraf": "4.4.0",
     "sinon": "15.0.2",
-    "source-map-loader": "4.0.1",
     "typescript": "5.1.6"
```

Comment:
@decentralgabe why were all of these dev dependencies removed?

Reply:
I ran `npx depcheck` and it told me to remove them - can revert if you want